tests/cases_homogenize.py

```diff
@@ -278,3 +278,62 @@ def case_rasterize_vector_success(
 )
 def case_rasterize_vector_fail(candidate_map, benchmark_map, rasterize_attributes):
     return candidate_map, benchmark_map, rasterize_attributes
+
+
+candidate_int32 = _load_xarray("candidate_map_0.tif")
```

Comment:
Do loading these outside of the cases function cause the data to persist across test cases and test functions?

Reply:
I put the loading dataset inside the case function